File: _basic_setup/java/processing_maven/src/main/java/org/example/Sketch.java
package org.example;

import processing.core.PApplet;

public class Sketch extends PApplet {
    @Override
    public void settings() {
        size(640, 360);
    }

    @Override
    public void setup() {
        windowTitle("CHANGE_ME");
        background(0); // black
    }

    @Override
    public void draw() {
        stroke(0, 255, 0); // borders green
        fill(255, 0, 0);
        circle(width / 2f, height / 2f, 20);
    }

}

File: _basic_setup/typescript/src/sketch.ts
import p5 from "p5";


new p5((p: p5) => {

  p.setup = () => {
    p.createCanvas(p.windowWidth, p.windowHeight);
    p.background(255); // white
  };

  p.draw = () => {
    p.stroke(0); // borders black
    p.circle(100, 100, 50);
    p.fill(255, 0, 0);
  };
});
